internal/storage/sql/db_test.go

```diff
@@ -36,12 +36,39 @@ func TestOpen(t *testing.T) {
 		{
 			name: "sqlite url",
 			cfg: config.DatabaseConfig{
-				URL:             "file:flipt.db",
+				URL:             "file:/flipt.db",
```

Comment:
is this backward compatible? ie does it also work with `file:flipt.db` ?

Reply:
@markphelps Yes it seems to work with `file:flipt.db` as well.

Reply:
can you add a test just to be sure?

Reply:
@markphelps Yup!